MANIFEST.in

```diff
@@ -1,2 +1 @@
 include README.md
-global-include *.typed
```

Comment:
@BryceStevenWilley looks like we need to patch upstream to stop this from happening in the Playground